Specs/Scene/ViewportQuadSpec.js

```diff
@@ -20,44 +17,34 @@ defineSuite([
         ClearCommand,
         Texture,
         Material,
-        createCamera,
-        createContext,
-        createFrameState,
-        pollToPromise,
-        render) {
+        createScene,
+        pollToPromise) {
     "use strict";
     /*global jasmine,describe,xdescribe,it,xit,expect,beforeEach,afterEach,beforeAll,afterAll,spyOn*/
 
-    var context;
-    var frameState;
+    var scene;
     var viewportQuad;
-    var us;
     var testImage;
 
     beforeAll(function() {
-        context = createContext();
-        frameState = createFrameState();
-
+        scene = createScene();
         return loadImage('./Data/Images/Red16x16.png').then(function(image) {
             testImage = image;
         });
     });
 
     afterAll(function() {
-        context.destroyForSpecs();
+        scene.destroyForSpecs();
     });
 
     beforeEach(function() {
         viewportQuad = new ViewportQuad();
         viewportQuad.rectangle = new BoundingRectangle(0, 0, 2, 2);
-
-        us = context.uniformState;
-        us.update(context, createFrameState(createCamera()));
+        scene.primitives.add(viewportQuad);
```

Comment:
I suggest writing
```
viewportQuad = new ViewportQuad();
scene.primitives.add(viewportQuad);
```
as
```
viewportQuad = scene.primitives.add(new ViewportQuad());
```
This code was written before `add` returned the added primitive.